### PR TITLE
Show audience selection in Sent Status TM

### PR DESCRIPTION
**Pull request by amyzhao1321**
Show audience selection in Sent Status TM

### CODE REVIEW

File: Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/StatusTaskModule/statusTaskModule.tsx [hunk 1]
```diff
@@ -142,6 +148,43 @@ class StatusTaskModule extends React.Component<RouteComponentProps, IStatusState
             );
         }
     }
+
+    private audienceSelection = () => {
```
Comment (jotrick) [inline Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/StatusTaskModule/statusTaskModule.tsx]: Change name to renderAudienceSelection

File: Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/StatusTaskModule/statusTaskModule.tsx [hunk 2]
```diff
@@ -142,6 +148,43 @@ class StatusTaskModule extends React.Component<RouteComponentProps, IStatusState
...
+                        } else {
+                            return (<span key={`teamName${index}`} >{team}, </span>);
+                        }
+                    })}
+                </div>);
+        } else if (this.state.message.rosterNames) {
+            let length = this.state.message.rosterNames.length;
+            return (
+                <div>
+                    <h3>Sent to General channel in teams</h3>
```
Comment (jotrick) [inline Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/StatusTaskModule/statusTaskModule.tsx]: Change string to "Sent in chat to people in teams"

File: Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/StatusTaskModule/statusTaskModule.tsx (hunk 1 above)
Comment (amyzhao1321) [reply Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/StatusTaskModule/statusTaskModule.tsx]: done

File: Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/StatusTaskModule/statusTaskModule.tsx (hunk 2 above)
Comment (amyzhao1321) [reply Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/StatusTaskModule/statusTaskModule.tsx]: done

**Pull request merged**